design/ceph/ceph-managed-disruptionbudgets.md

```diff
@@ -24,16 +24,16 @@ removed from the node in the event of a drain, we will rely on the assumption th
 as false positives for drains are not dangerous in this use case.
 
 Example flow:
-- A Ceph pool CRD is created.
```

Comment:
I suggest that we not make formatting changes to the design docs. Changing the bullets to a numbered list doesn't change the readability much. This type of thing is fine to change in documentation, but let's not worry about the formatting as much in the design docs.

Reply:
OK, I'll remove the changes of design docs.